bedrock/firefox/templates/firefox/choose.html

```diff
@@ -33,7 +33,13 @@ <h2 data-tracking="{{ _('Browse Freely.<br> Choose Firefox.') }}" data-choice="{
 <section class="page-content">
   <div class="blurb">
     <h2>{{ _('The most private browsing') }}</h2>
-    <p>{{ _('Take control of your web by using Private Browsing with Tracking Protection, only from Firefox.') }}</p>
+    <p>
+      {% if LANG == 'en-US' %}
```

Comment:
Could show this string for any locale that starts with `en-` instead of just `en-US`?